app/controllers/api/mentoring/discussions/previous_sessions_controller.rb

```diff
@@ -0,0 +1,24 @@
+module API
```

Comment:
@iHiD I decided to let the server handle this instead of the excluding the current discussion from the client. If we just exclude the current discussion, it will be hard for us to recreate the correct pagination.